src/IssueViz.UnitTests/Editor/ErrorTagging/ErrorTaggerTests.cs

```diff
@@ -42,17 +42,25 @@ public void GetTags_FilterIsApplied_ExpectedTagsCreated()
             var inputSpans = CreateSpanCollectionSpanningWholeSnapshot(snapshot);
```

Comment:
I added to more tag spans that are both suppressed however now that I am looking at it it feels a bit ugly that I do not set the others to NotSuppressed. What are your thoughts? Also is this adequate test coverage in your opinion?

Reply:
You could add an optional property `bool isSuppressed = false` to the helper methods `CreateTagSpanWithPrimaryLocation` and `CreateIssueViz`.

That would also avoid the ugly casting in lines 59 - 62 to set the `IsSuppressedProperty`.